src/reducers/flexColumnWidths.js

```diff
@@ -40,7 +40,7 @@ export function initializeFlexColumnWidths(state) {
   while (availableWidth > 0 && idx < scrollableColumnsCount) {
     const column = getScrollableColumn(state, idx);
     availableWidth -= column.props.width;
-    flexTotal += column.props.flexGrow;
+    flexTotal += column.props.flexGrow || 0;
```

Comment:
In case flexGrow is not defined flexTotal evaluate to Nan, fixes that alongwith.

Reply:
good catch